src/main/java/com/bytebreak/animagic/Animator.java

```diff
@@ -18,9 +18,10 @@ public Animator(@NotNull String name){
         this.name = name;
     }
 
-    public void addAnimation(@NotNull IFrameByFrameAnimation animation){
+    public Animator addAnimation(@NotNull IFrameByFrameAnimation animation){
         if (animation == null) throw new AnimagicException(name + ": Cannot add a null animation");
         animations.add(animation);
+        return this;
```

Comment:
I think this is more of a Builder pattern thing, yeah? Libgdx is the first place I saw do this --- I like it a lot. Chaining is super nice.